docs/kpod-logout.1.md

```diff
@@ -0,0 +1,49 @@
+% kpod(1) kpod-logout - Simple tool to logout of a registry server
+% Urvashi Mohnani
+# kpod-logout "1" "August 2017" "kpod"
+
+## NAME
+kpod-logout - Logout of a container registry
+
+## SYNOPSIS
+**kpod logout**
+**REGISTRY**
+[**--help**|**-h**]
+
+## DESCRIPTION
+**kpod logout** logs out of a specified registry server by deleting the credentials
+stored in the **auth.json** file. The path of the authentication file can be overrriden by the user by setting the **infile** flag. The default path used is **XDG\_RUNTIME\_DIR/containers/auth.json**.
+
```

Comment:
I think you should add
${XDG_\RUNTIME_DIR}

Reply:
fixed